README.md

```diff
@@ -0,0 +1,23 @@
+# Arcadia Science Software Handbook
+
+The repository is home to the core software engineering guides and policies at [Arcadia Science](https://www.arcadiascience.com/). If you're curious about where to begin, [the source control policy document](/guides-and-policies/policy-source-control) is a good place to start!
+
+## Naming convention
+
+- A **guide** document describes software engineering best practices for reusable and reproducible work. These are strong suggestions but exceptions to the rule can happen.
```

Comment:
strong suggestions vs. sensible defaults? Curious how strongly we want to push toward things written in guides.

Reply:
I like sensible defaults!